browser/ui/views/download/brave_download_item_view.cc

```diff
@@ -55,6 +56,14 @@ BraveDownloadItemView::~BraveDownloadItemView() {}
 
 // View overrides.
 
+void BraveDownloadItemView::Layout() {
```

Comment:
please cc the owner of `DrawStatusText`

Reply:
That's me. I will add an issue to transition our code to use a label as well.